data/oh/legislature/Darrell-Kick-92de07aa-a6c7-4c4b-ada7-02ebbe4a17f6.yml

```diff
@@ -2,7 +2,7 @@ id: ocd-person/92de07aa-a6c7-4c4b-ada7-02ebbe4a17f6
 name: Darrell Kick
 given_name: Darrell
 family_name: Kick
-email: rep70@ohiohouse.gov
+email: rep98@ohiohouse.gov
 biography: State Representative Darrell Kick is serving his second term in the Ohio
```

Comment:
@jessemortenson we may want to remove this bio which indicates he represents the 70th district, because he now represents the 98th district.